src/CMake/FindConduit.cmake

```diff
@@ -49,4 +57,5 @@ if(DEFINED CONDUIT_VERSION)
         set(CONDUIT_HAVE_PARTITION_FLATTEN 1)
     endif()
 endif()
+
```

Comment:
Is the above logic to allow for building with older conduit still necessary? Who is building VisIt that way still?

Reply:
Until we fully update TPLs on LC this could be a barrier.